packages/backend/src/graphql/mutations/tiles/create-row.ts

```diff
@@ -0,0 +1,33 @@
+import { IJSONPrimitive } from '@plumber/types'
+
+import { createTableRow } from '@/models/dynamodb/table-row'
+import Context from '@/types/express/context'
+
+type Params = {
+  input: {
+    tableId: string
+    data: Record<string, IJSONPrimitive>
+  }
+}
+
+const createRow = async (
+  _parent: unknown,
+  params: Params,
+  context: Context,
+) => {
+  const { tableId, data } = params.input
+  const table = await context.currentUser
+    .$relatedQuery('tables')
+    .findById(tableId)
+    .throwIfNotFound()
+
+  if (!(await table.validateRowKeys(data))) {
```

Comment:
nit: if you use `withGraphFetched` on the initial query you can potentially save a query here. You can maintain the abstraction by passing `table` into `validateRowKeys`

Reply:
Can do funny thing like this

```
const table = await context.currentUser
    .$relatedQuery('tables')
    .withGraphFetched('columns(onlyColumnId)')
    .modifiers({
      onlyColumnId(builder) {
        builder.select('id')
      },
    })
    .findById(tableId)
    .throwIfNotFound()
```